### ISSUE TITLE
Problem with Point Instruction on Amstrad CPC

### ISSUE THREAD

**Issue by ldir-hector** [spotlessmind1975/ugbasic#593]
the function POINT (x,y) have some problems I illustrate with the following program. It behaves differently when adding some simple instructions.

```
BITMAP ENABLE(160,200,16)
CLS RGB(255,0,0)
INK YELLOW
PLOT SCREEN WIDTH / 2, SCREEN HEIGHT / 2, RGB(0,255,0)
PLOT (SCREEN WIDTH / 2)+1, SCREEN HEIGHT / 2, RGB(0,255,0)
PEN WHITE
PRINT POINT(SCREEN WIDTH / 2, SCREEN HEIGHT / 2)

PRINT POINT(SCREEN WIDTH / 2, (SCREEN HEIGHT / 2)+2)
'this one is incorrect I shall have 12 but I got 245 
'(which if I do mod 27 gives 2 which is incorrect too !)
'The tricky part, is that before adding the fourth test I got
'201 which mod 27 gives 12 which is correct.
'But, if I remove the third test I got 193 which mod 27 gives    
'4 that is incorrect.
'There are much side effects on this function.    
PLOT 120,140,RGB(0,0,0)
PRINT POINT(120,140)
'this test is incorrect shall be 20 and I got 25

'PLOT 120,150,RGB(0,0,255)
'PRINT POINT(120,150) 
'this test is correct
    
HALT
  
```
I am assuming that POINT(x,y) returns the color from the CRTC palette : 
![image](https://user-images.githubusercontent.com/118670680/230611684-2dc79c41-d870-40b6-afde-d1d9764cbcc8.png)
and that if the number is beyond 27, the user must do a modulo 27. It seems to work, but not all the times and the fact that the results change if I remove some of the tests, is confusing !

**Comment by spotlessmind1975** [comment]
Hello @ldir-hector , the problem has been fixed on COLDFIX v1.13.2 [rev. 20230408.1348].

**Comment by spotlessmind1975** [comment]
Ok, NOW it is fixed in COLDFIX v1.13.2 [rev. 20230408.1440].
